src/test/java/page/GoogleHomePage.java

```diff
@@ -0,0 +1,36 @@
+package page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class GoogleHomePage extends BasePage {
+
+    private static final String GOOGLE_HOMEPAGE_URL = "https://www.google.com/";
+    private WebDriver driver;
+
+    @FindBy(name = "q")
+    private WebElement googleSearchField;
+
+    @FindBy(name = "btnK")
+    private WebElement googleSearchButton;
+
+    public GoogleHomePage(WebDriver driver) {
+        this.driver = driver;
+
+        PageFactory.initElements(driver, this);
+    }
+
+    public GoogleHomePage openGooglePage() {
+        pageOpener(GOOGLE_HOMEPAGE_URL);
+        return this;
+    }
+
+    public GoogleSearchForWikipediaResultsPage searchForTerms(String searchTerm) {
+        pageOpener(GOOGLE_HOMEPAGE_URL);
```

Comment:
this line need to be removed from this method. Your method is about Search but not about Opening Pages.

Reply:
Got it. Removed to BaseTest erased a redundant code